build.sc

```diff
@@ -890,6 +890,15 @@ object contrib extends MillModule {
     )
   }
 
+  object jmh extends MillModule {
```

Comment:
```suggestion
  object jmh extends MillInternalModule with AutoTestSetup with WithMillCompiler {
```

This might help to avoid the failing MiMa checks.